cpp/oneapi/dal/algo/linear_regression/common.cpp

```diff
@@ -60,14 +95,25 @@ namespace v1 {
 using detail::v1::model_impl;
 
 template <typename Task>
-model<Task>::model() : impl_(nullptr) {}
+model<Task>::model() : impl_{ std::make_shared<detail::model_impl<Task>>() } {}
 
 template <typename Task>
-model<Task>::model(const std::shared_ptr<detail::model_impl<Task>>& impl) : impl_(impl) {}
+model<Task>::model(const std::shared_ptr<detail::model_impl<Task>>& impl) : impl_{ impl } {}
 
 template <typename Task>
 const table& model<Task>::get_betas() const {
-    return impl_->get_betas();
+    return impl_->get_packed_coefficients();
+}
+
+template <typename Task>
+const table& model<Task>::get_packed_coefficients() const {
```

Comment:
Should we also extend the docs as new API methods were added?
BTW, I do not see the docs for linear regression at all in github.io.

Reply:
Just because there are no such docs for all new OneAPI algorithms at all. And yes it is in todo list but not with a low priority

Reply:
Discussed